src/pages/CommandView/commandViewHelpers.tsx

```diff
@@ -0,0 +1,244 @@
+import { Box, Button, Typography } from '@mui/material'
+import { useNavigate } from 'react-router-dom'
+import { Parameter } from 'types/backend-types'
+import {
+  AugmentedCommand,
+  ObjectWithStringKeys,
+  StrippedSystem,
+} from 'types/custom-types'
+
+interface NavigatorProps {
+  message: string
+  link: string
+}
+/* This is an ugly placeholder */
```

Comment:
This just slaps a button on the page whenever the context is found to be out of whack. As the comment implies, this is not meant to be the final version of this.